src/swift_server.rs

```diff
@@ -185,6 +190,36 @@ pub async fn process_order(
                 }),
             );
         }
+    };
+
+    if user.is_none() {
+        log::warn!(
+            target: "server",
+            "{log_prefix}: Error simulating order, user not found"
+        );
+        return (
+            axum::http::StatusCode::BAD_REQUEST,
+            Json(ProcessOrderResponse {
+                message: PROCESS_ORDER_RESPONSE_ERROR_USER_NOT_FOUND,
+                error: Some("user not found".to_string()),
+            }),
+        );
+    }
+
+    // If fat fingered order that requires sanitization, then just send the order
+    let mut order_params = order_params.clone();
+    if server_params
+        .simulate_will_auction_params_sanitize(&mut order_params)
+        .await
+    {
+        /// TODO: Send the transaction here
```

Comment:
does it makes sense for another process to handle the tx submission? e.g. send via a different kafka topic

can add a RetryTxSender in drfit-rs.
there's an example here:
https://github.com/drift-labs/gateway/blob/1c4853fd5d9d8e092e607a95915b24700f6e390f/src/controller.rs#L648-L653